### PR TITLE
[docs] #116: Add openssl installation instructions

### LINKED ISSUE #116: Add the openssl installation to the instruction.
Build guide doesn't work because we don't have openssl installation command.

### PR DESCRIPTION
**Pull request by outoftardis**
Closes #116

Signed-off-by: Ekaterina Mekhnetsova <mekkatya@gmail.com>

### CODE REVIEW

File: src/guide/build-and-install.md [hunk 1]
```diff
@@ -178,6 +179,27 @@ then the issue runs deeper.
...
+- Instal OpenSSL on Ubuntu:
+
+  ```bash
+  sudo apt-get install libssl-dev
+  ```
+
+- Instal OpenSSL on macOS:
+
+  ```bash
+  brew install openssl
```
Comment (astrokov7) [inline src/guide/build-and-install.md:196]: I'm not sure that the user will have brew installed, because we haven't mentioned it before.

Comment (outoftardis) [reply src/guide/build-and-install.md:196]: linked to to the docs where users can find the install command and other info

File: src/guide/build-and-install.md [hunk 2]
```diff
@@ -178,6 +179,27 @@ then the issue runs deeper.
...
+Make sure you have OpenSSL installed. Note that in most Linux setups it is
+already available to you.
+
+- Install OpenSSL on Ubuntu:
+
+  ```bash
+  sudo apt-get install libssl-dev
+  ```
+
+- Install OpenSSL on macOS using [`brew`](https://brew.sh/):
```
Comment (astrokov7) [inline src/guide/build-and-install.md:193]: brew it is displayed not as a link, but as a code.
image

Comment (outoftardis) [reply src/guide/build-and-install.md:193]: It's both a link and the inline code
I can make it just a link if you think it's better

**Review by astrokov7 — APPROVED** [review]

**Pull request merged**